source/compute/portal/ondemand/desktop.rst

```diff
@@ -36,3 +43,86 @@ For improved graphics performance, we recommend the following workflow:
    (Share-able Link)' button to copy the link and share it with others. Any VSC
    user with the link can view your Desktop session in real-time, which is
    especially helpful for job troubleshooting and hands-on training.
+
+.. tip::
+
+   In certain browsers (e.g. Google Chrome), the clipboard in the Desktop app
+   will have the same content as your device, meaning that you can copy-paste
+   text to and from the Desktop as expected. Other browsers, like Firefox,
+   do not currently support this and in that case you will need to transfer
+   text through the clipboard in the panel on the left hand side.
+
+.. tip::
+
+   The Applications menu on the top left will offer a number of shortcuts.
+   Note that some may be missing in case you have changed the value of the
+   ``XDG_DATA_HOME`` environment variable in your ``~/.bashrc`` to something
+   other than ``$VSC_HOME/.local/share``.
+
+
+Limitations specific to KU Leuven
+---------------------------------
+
+Mainly due to security concerns, the KU Leuven OOD Desktop app is run within
+a minimalistic container. As a result some functionality is deliberately
+missing:
+
+- Slurm-related commands such as ``squeue``, ``sbatch``, ``srun``, ...
+  and ``sam-balance``, ``sam-statement``, ... are unavailable.
+
+- No browser is provided within the Desktop app (see :ref:`the subsection below
+  <ood_desktop_local_browser_tips>` for alternatives).
+
+Some applications are currently unavailable but may be enabled in the future
+(such as ``myquota``, ``apptainer`` and archive managers such as
+``xarchiver``).
+
+
+.. _ood_desktop_local_browser_tips:
+
+Using your local browser where needed
+=====================================
+
+Certain applications expect that you have a browser at your disposal.
+
+The more common case is where the application starts a server to which
+you are expected to connect with a browser (examples: JupyterLab,
```

Comment:
JupyterLab should preferably be used as a dedicated app, so i wouldn't mention it here.